warehouse/models/mart/gtfs/_mart_gtfs_fcts.yml

```diff
@@ -931,7 +933,12 @@ models:
         description: |
           Synthetic primary key constructed from `service_alerts_message_key`,
           `agency_id`, `trip_id`, `route_id`, and `stop_id`.
-        tests: *rt_primary_key_tests
+        tests: &service_alert_key_tests
+          - unique_proportion:
+              at_least: 0.98
```

Comment:
Why is this proportion noticeably lower?

Reply:
I assume because service alerts are more likely to be human-generated and therefore violate our uniqueness assumptions.

Reply:
They were about 98.5% unique but most of the others were at minimum 99%